dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/utils/ArchiveNameUtil.java

```diff
@@ -36,6 +40,17 @@ public static String getFileNameWithTimestamp(String name, Clock clock) {
     return formatFileName(name, timeSuffix);
   }
 
+  public static String getEntryFileNameWithTimestamp(String prefix, ZonedInterval interval) {
+    return getEntryFileNameWithTimestamp(prefix, interval, ZIP_ENTRY_SUFFIX);
+  }
+
+  public static String getEntryFileNameWithTimestamp(
+      String prefix, ZonedInterval interval, String suffix) {
+    DateTimeFormatter formatter =
```

Comment:
I think `formatter` can be `static final`.

Reply:
I moved it together with second formatter used for archive name.